ocr_service/tests/api/test_ocr_endpoints.py

```
@@ -0,0 +1,17 @@
+import pytest
```

Ruff (0.8.2)

`pytest` imported but unused

Remove unused import: `pytest`

(F401)